dll/win32/shell32/lang/es-ES.rc

```diff
@@ -332,7 +332,7 @@ BEGIN
     EDITTEXT 14008, 197, 103, 33, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
     CONTROL "", 14015, "Static", SS_OWNERDRAW, 70, 116, 100, 30
     LTEXT "Unidad %c", 14009, 100, 150, 70, 10
-    PUSHBUTTON "&Liberar espacio en disco", 14010, 160, 154, 70, 15, WS_TABSTOP
+    PUSHBUTTON "&Liberar espacio en disco", 14010, 143, 152, 88, 15, WS_TABSTOP
```

Comment:
it was too large?

Reply:
The button was smaller than the text. And some of them couldn't be seen.